runexample

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Please remove this bat, and document the content properly in README.

As we also support windows, we don't to introduce an example which can only run on linux.